flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/fs/ByteBufferWritableFSDataOutputStream.java

```diff
@@ -104,4 +109,8 @@ public void sync() throws IOException {
     public void close() throws IOException {
         originalOutputStream.close();
     }
+
+    public Path getRealPath() {
```

Comment:
Well I think we need this interface in `ForStFlinkFileSystem`. It is useful during checkpoint.

Reply:
Added a static function in `ForStFlinkFileSystem`.